javascript-data-model/main.js

```diff
@@ -0,0 +1,56 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    orderplaced: "August 4, 2020",
+    total: 34,
+    price: 31.55,
+    shipto: "JS Masher",
+    ordernumber: "114-3941689-8772232",
+    delivered: "August 8, 2020"
+    title: "Javascript for impatient programmers",
+    author: "Rauschmayer, Dr. Axel"
+  },
+  {
+    orderplaced: "July 19, 2020",
+    total: 44.53,
+    price: 41.33,
+    shipto: "JS Masher",
+    ordernumber: "113-9984268-1280257",
+    delivered: "July 20, 2020",
+    title: "The Timeless Way of Building",
+    author: "Alexander, Christopher"
+  },
+  {
+    orderplaced: "July 4, 2020",
+    total: 17.22,
+    price: 15.98,
+    shipto: "JS Masher",
+    ordernumber: "114-2875557-9059409",
+    delivered: "July 7, 2020",
+    title: "Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter",
+    author: undefined
+  },
+  [
```

Comment:
I would recommend using a single object here for the order, and then an array for one of the properties which contains the two objects which represent the items ordered